src/app/Main/Body/QueryPage/ChangeDisplFeatures.service.tsx

```diff
@@ -0,0 +1,12 @@
+import { ValueType } from "react-select";
+import { FilterType } from "../../../Shared/Model/Filter.model";
+
+export function handleChangeDisplayedFeatures(
+    selectedOption: ValueType<{value: string, label: string}>
+): FilterType {
+    if (selectedOption) {
+        const selectedFeature: string = (Object.values(selectedOption)[0]);

```

Comment:
I already tried different methods to validate the data in this case. 
Your idea doesn't work because "ValueType" is a type of the selector. Which works equally for the "displayed features" ({ value: string; label: string; }) and for the "filters" (OptionsType<{ value: string; label: string; }>). 

**Property 'value' does not exist on type '{ value: string; label: string; } | OptionsType<{ value: string; label: string; }>'.
  Property 'value' does not exist on type 'OptionsType<{ value: string; label: string; }>'.**

Reply:
I could type the selectedOption like you suggested in the following comment for the ChangeFilter.service.ts. 

`const selectedOptionObj = selectedOption as {value: string, label: string}`

Reply:
Done. Commit: "Adapt types of selector onChange"

Reply:
partly done: just converting the type does not do the actual check if the type is the one you expect, see later review

Reply:
Done